lib/l10n/intl_ro.arb

```diff
@@ -294,11 +304,16 @@
   "infoReadThePolicy": "Citește politica {appName}",
   "infoExportToGoogleCalendar": "Exportă evenimentele filtrate din Orar",
   "infoFormAnonymous": "Acest formular este anonim.",
+  "infoFeedbackForm": "Oferiți feedback și raportați probleme",
 
   "stringEmailDomain": "@stud.acs.upb.ro",
   "stringForum": "cs.curs.pub.ro",
   "stringAnonymous": "Anonim",
   "stringAnd": "și",
 
-  "fileAcsBanner": "assets/images/acs_banner_ro.png"
+  "fileAcsBanner": "assets/images/acs_banner_ro.png",
+
+  "emailExample": "exemplu: john.doe123@gmail.com",
+  "feedbackExample": "Scrieți-vă feedback-ul",
+  "issueExample": "Descrieți problema"
```

Comment:
Wouldn't it be better to keep the old ones and just replace the one for email with something like "Enter your email/phone number/etc." since we changed that field to "Contact info"? Just so we keep the advice with the actions that the user can do

Reply:
it's kind of inconsistent with other fields in our app where we actually have an example, so I'd rather keep the example. I don't think the user doesn't understand what a feedback form is xD